python/tank/util/shotgun.py

```diff
@@ -213,7 +213,7 @@ def _raise_missing_key(key):
     return config_data
 
 @LogManager.log_timing
-def download_url(sg, url, location):
+def download_url(sg, url, location, use_url_extension=False):
```

Comment:
missing docstring for `use_url_extension`. Please examples of this parameter being false and true to the docstring - i think that's useful for users!